x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -14,36 +14,31 @@ export const exceptionsArtifactSavedObjectType = ArtifactConstants.SAVED_OBJECT_
 export const manifestSavedObjectType = ManifestConstants.SAVED_OBJECT_TYPE;
 
 export const exceptionsArtifactSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
```

Comment:
So this SO type (`endpoint:user-artifact`) is actually one we no longer use and have an issue open to remove it (team issue 6214), so whatever changes you want to do here, you can

Reply:
@paul-tavares I went ahead and removed the SO here: https://github.com/elastic/kibana/pull/155204/commits/19d06c7c3ee1ad8db562d8e5702ba57738f72597